README.md

```diff
@@ -202,6 +202,13 @@ The full Matterport3D (MP3D) dataset for use with Habitat can be downloaded usin
 #### Gibson
 Download the Habitat related Gibson dataset following the instructions [here](https://github.com/StanfordVL/GibsonEnv#database). After downloading extract the dataset to folder `habitat-lab/data/scene_datasets/gibson/` folder (this folder should contain the `.glb` files from Gibson).
 
+#### Habitat test scenes
+Download the scenes used for performing unit tests via the download api from [habitat-sim](https://github.com/facebookresearch/habitat-sim).
+
+```
+python -m habitat_sim.utils.datasets_download --uids habitat_test_scenes
+```
+
```

Comment:
I don't want to expand the scope of your PR too much, but I think we should kill this entire section in the README (from ReplicaCAD to Habitat test scenes) and simply refer people to Habitat-Sim/Dataset.md. This will avoid duplication.

Reply:
I can take a stab at this, sure.

Reply:
@dhruvbatra  - What do you think about the latest commit? I can also remove the pointnav dataset download instructions from habitat-sim/DATASETS.md (see below) since it isn't very relevant there.

https://github.com/facebookresearch/habitat-sim/pull/1821/files#diff-2f3570bea2b91e76817d6b45586c34747ad6b9f463bb8563d85c519fb7e36c29R11-R16